tasks/src/tabular-regression/about.md

```diff
@@ -12,6 +26,13 @@ Tabular regression is the task of predicting a numerical value given a set of at
 
 ## Useful Resources
 
+- [Kaggle Tabular Playground Series](https://www.kaggle.com/search?q=tabular+playground+in%3Acompetitions+sortBy%3Adate+competitionEvaluationAlgorithm%3ARMSE+competitionEvaluationAlgorithm%3ASMAPE+competitionEvaluationAlgorithm%3AMAE+competitionEvaluationAlgorithm%3AMCRMSLE) comes up with a new tabular challenge every month.
+- Tabular Regression using [FastAI](https://walkwithfastai.com/Regression_and_Permutation_Importance).
+
+- [Data Science for tabular data: Advanced Techniques](https://www.kaggle.com/code/vbmokin/data-science-for-tabular-data-advanced-techniques/notebook)
```

Comment:
This seems like a compilation of winning solutions for challenges, which is not really good for the audience of these pages (sort of beginner to intermediate)

Reply:
Actually the first one is just a list of kaggle tabular playground series competition, and the rest are tutorial using FastAI and last one is advanced tips however I understand is required. I will modify this in _Beginner to Intermediate_ format.

Reply:
The line I'm indicating is this: 
`- [Data Science for tabular data: Advanced Techniques](https://www.kaggle.com/code/vbmokin/data-science-for-tabular-data-advanced-techniques/notebook)`

Reply:
Ohh okay.